lsec2

```diff
@@ -62,6 +65,7 @@ def main():
 
     # get list of instances
     instances = ec2.instances.filter(Filters=[{'Name': 'instance-state-name', 'Values': ['running']}])
+    instances = instances.filter(Filters=[{'Name': 'tag:Name', 'Values': ['*' + filter + '*']}])
```

Comment:
**Suggestion:** It's a good practice to use f-strings for string formatting in Python 3.6 and above. It's more readable and efficient.
```suggestion
    instances = instances.filter(Filters=[{'Name': 'tag:Name', 'Values': [f'*{filter_arg}*']}])
```